app.js

```diff
@@ -1,17 +1,67 @@
 var express = require('express');
 var databaseConfig = require('./config/databaseConfig');
+var flash = require('connect-flash');
+var passport = require('passport'),
+  LocalStrategy = require('passport-local').Strategy;
```

Comment:
PassportJS module and the local strategy module used for authenticating with username and password.